extractors/paths/url_extraction/config.py

```diff
@@ -15,8 +15,12 @@ def get_config():
         available_for=["refinery", "common"],
         part_of_group=[
             "paths",
+            "personal_identifiers"
         ],  # first entry should be parent directory
         # bricks integrator information
+        cognition_init_mapping={
+            "@@LABEL@@": "URL"
+        },
```

Comment:
same as above, URL wasn't in the initial list of labels but i can add it if needed.